packages/nc-gui/pages/index/index/index.vue

```diff
@@ -39,9 +39,17 @@ const filterQuery = ref('')
 
 const projects = ref<ProjectType[]>()
 
+const activePage = ref(1)
+
+const pageChange = (p: number) => {
+  activePage.value = p
+}
+
 const loadProjects = async () => {
+  const lastPage = activePage.value
   const response = await api.project.list({})
   projects.value = response.list
+  activePage.value = lastPage
```

Comment:
would `activePage.value` be changed after project list? Besides, I think it would be better to name `lastActivePage` or `prevActivePage` as `lastPage` sounds like the last page of the table.